cluster/src/main/scala/io/snappydata/remote/interpreter/RemoteInterpreterStateHolder.scala

```diff
@@ -221,7 +229,8 @@ class RemoteILoop(spw: StringPrintWriter, intpHelper: RemoteInterpreterStateHold
 
 object RemoteILoop {
   private val notTBeInheritedCommandNames = Set(
-    "h?", "edit", "line", "load", "paste", "power", "quit", "replay", "reset", "settings")
+    "h?", "edit", "line", "load", "paste", "power",
```

Comment:
Providing an examples for loading external scala file would be nice? 
Also, are all these commands documented in Spark guides ?